src/test/java/org/broadinstitute/hellbender/utils/fasta/CachingIndexedFastaSequenceFileUnitTest.java

```diff
@@ -39,10 +42,13 @@
     @DataProvider(name = "fastas")
     public Object[][] createData1() {
         List<Object[]> params = new ArrayList<>();
-        for ( Path fasta : Arrays.asList(simpleFasta) ) {
+        /** the tests compare to {@link simpleFasta} as the uncached version of the file
+         * you can't add arbitrary fastas here without refactoring
+         */
```

Comment:
I like to use javadoc comments because they let you include links to other methods and the ide will warn you if they're out of date.  Can change though.